### PR TITLE
Implemented Unit test

### PR DESCRIPTION
**Pull request by akkuldn**
implemented the unit test and updated the requirements file

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,43 +1,72 @@
-alembic==1.1.0
+apipkg==1.5
+Appium-Python-Client==0.28
```
Comment (qxf2) [inline requirements.txt]: This entire requirements file looks wrong. Where are we using appium or results or arrow in this project?

File: tests/test_mock_base_gcal.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    """
+    #date that is a qxf2 holiday
+    date='03/04/2019'
+    
+    #Hardcoding the date in datetime format
+    time_mock=datetime.datetime.strptime(date,'%m/%d/%Y')
+    
+    #mocking gcal.process_date_string method
+    @mock.patch('qxf2_scheduler.qxf2_scheduler.gcal.process_date_string',return_value=time_mock)      
+
```
Comment (qxf2) [inline tests/test_mock_base_gcal.py]: There should be no empty line between the decorator and the method definition

File: tests/test_mock_base_gcal.py [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    def test_is_qxf2_holiday(self, mockbase_gcal):
```
Comment (qxf2) [inline tests/test_mock_base_gcal.py]: Please select a better method name. There can be many tests for the is_holiday method. So naming something test_is_, holiday is not very descriptive.

File: tests/test_mock_base_gcal.py [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+        """
+        This method tests wether the function displays True if the given date is a qxf2 holiday
+        """
+        assert is_qxf2_holiday(self.date)==True
+
+    #mocking gcal.process_date_string method   
+    @mock.patch('qxf2_scheduler.qxf2_scheduler.gcal.process_date_string',return_value=time_mock) 
+    
+    def test_parameters(self,mockbase_gcal):
```
Comment (qxf2) [inline tests/test_mock_base_gcal.py]: Method name needs to change. When I see a method name, I should know what it does. Test_parameters is too generic.

File: requirements.txt [hunk 5]
```diff
@@ -1,72 +1,48 @@
...
 docopt==0.6.2
-docutils==0.14
-execnet==1.7.0
 Flask==1.1.1
+Flask-Migrate==2.5.2
+Flask-Script==2.0.6
 Flask-SQLAlchemy==2.4.0
-future==0.17.1
-google-api-python-client==1.7.11
+google-api-python-client==1.7.9
```
Comment (qxf2) [inline requirements.txt]: Why did this a couple of minor versions lower?

**Comment by akkuldn** [comment]
I have made the necessary changes and am creating a new pull request ,therefore im closing this previous request

**Pull request closed without merging**